sparse_ho/algo/implicit.py

```diff
@@ -81,25 +81,25 @@ def get_beta_jac_t_v_implicit(
     X: array-like, shape (n_samples, n_features)
         Design matrix.
     y: ndarray, shape (n_samples,)
-        observation vector.
+        Observation vector.
     log_alpha: float or np.array, shape (n_features,)
-        logarithm of hyperparameter.
+        Logarithm of hyperparameter.
     mask0: ndarray, shape (n_features,)
-        boolean of active active for warm start
+        Boolean of active for warm start
     dense0: ndarray, shape (n_features,)
-        initial value of the value on the support for warm start
+        Initial value of the value on the support for warm start.
     tol: float
-        the tolerance for the optimization
+        The tolerance for the optimization.
```

Comment:
for the inner optimization pb may be clearer?

Reply:
done